extensions/rich_text_components/Collapsible/Collapsible.html

```diff
@@ -1,4 +1,10 @@
 <script type="text/ng-template" id="richTextComponent/Collapsible">
+  <style>
+    .conversation-skin-tutor-card-top-content.protractor-test-conversation-content {
```

Comment:
The main issue with the `.conversion-skin-tutor-card-top-content` class is whether or not it is okay for all accordions to be the full width of the card. I don't see this being a problem as the current behavior of accordions is to not allow other content on the same line as them, but as it will change all accordion's appearance I would want to make sure that this side-effect is okay.

<img width="603" alt="screen shot 2016-03-01 at 2 59 47 pm" src="https://cloud.githubusercontent.com/assets/13967323/13440286/50904116-dfbf-11e5-95f3-d62b899f655e.png">
*With my current changes a video in an accordion makes all other accordions full width in that card*

<img width="614" alt="screen shot 2016-03-01 at 2 59 57 pm" src="https://cloud.githubusercontent.com/assets/13967323/13440287/5090f570-dfbf-11e5-998a-9fc5d4855e29.png">
*Default width of accordion elements that are dependent on the content size*